docs/developing/dev_environment/local_proxy_remote_solana.md

```diff
@@ -1,68 +1,76 @@
 ---
 title: "Local Proxy to Remote Solana"
-proofedDate: na
+proofedDate: 20230828
 iterationBy: na
 includedInSite: true
 approvedBy: na
-comment: 
+comment: Local to local was updated -- todo verify this page; may need update also e.g. instruction says For Devnet/Testnet, it should be `eeLSJgWzzxrqKv1UxtRVVH8FX3qCQWUs9QuAjJpETGU` but are Devnet and Testnet not two different entities?
```

Comment:
I did a light pass over this item to proof it and there are a couple of potential issues with the remaining page that triggered me @YYYurchenko -- this item in the comment where 1 setting apparently controls for 2 testnets and the final line -- whereby folk that are not registered as Operators need to get public keys. Do I need to open a ticket to review the veracity of this page and whether it is possible to complete the tutorial as it stands?